src/core/hle/libraries/libkernel/file_system.cpp

```diff
@@ -1,6 +1,12 @@
-#include "common/log.h"
```

Comment:
Headers are included in alphabetical order, which means common headers are always first.

Reply:
seems that bundled clang does it differently sorting